src/binary/Builtin.cc

```diff
@@ -632,6 +632,39 @@ extern "C" {
     return INT2FIX(strcoll(RSTRING_PTR(str1), RSTRING_PTR(str2)));
   }
 
+  static struct tm *
+  hash_to_tm(VALUE hash)
+  {
+    struct tm res;
+
+    res.tm_sec    = rb_hash_aref(hash, ID2SYM(rb_intern("tm_sec")));
```

Comment:
Indeed. And that explain why I was getting "dic 51 37:73:103" instead of "jun 25 18:41:19" :-)